client/state/ui/drop-zone/reducer.js

```diff
@@ -17,10 +13,16 @@ import {
 
 import { createReducer } from 'state/utils';
 
-const isVisible = createReducer( false,
+const isVisible = createReducer( {},
 	{
-		[ DROPZONE_SHOW ]: stubTrue,
-		[ DROPZONE_HIDE ]: stubFalse,
+		[ DROPZONE_SHOW ]: ( state, { dropZoneName } ) => ( {
+			...state,
+			[ dropZoneName ]: true,
+		} ),
+		[ DROPZONE_HIDE ]: ( state, { dropZoneName } ) => ( {
+			...state,
+			[ dropZoneName ]: false,
+		} ),
```

Comment:
I didn't explain properly. Some brief context: `createReducer` allows a `schema` to be passed as third arg; if it's present, it will be used for state validation upon a `DESERIALIZE`. If not, both `SERIALIZE` and `DESERIALIZE` will yield `initialState`.

Because of this, it could be expected that this behavior be kept with `keyedReducer`, but it doesn't. Notably, if an action going through the reducer made by `keyedReducer` doesn't have a property matching `key`, the reducer will return `state` — i.e. the state will be kept. `{DE,}SERIALIZE` actions contain no such properties, so they will yield no state changes, when one might prefer that they yield `initialState`.